pkgs/development/python-modules/ml-dtypes/default.nix

```diff
@@ -27,9 +29,18 @@ buildPythonPackage rec {
     fetchSubmodules = true;
   };
 
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace 'numpy~=' 'numpy>=' \
+      --replace 'pybind11~=' 'pybind11>=' \
+      --replace 'setuptools~=' 'setuptools>='
+  '';
```

Comment:
why is this necessary/an improvement? in other words, if it ain't broke why fix it?

Reply:
This will break in `staging` now since https://github.com/NixOS/nixpkgs/pull/248866 was merged.